VENCore.podspec

```diff
@@ -1,19 +1,14 @@
 Pod::Spec.new do |s|
-  s.name         = "VENCore"
-  s.version      = "3.1.1"
-  s.summary      = "Core Venmo client library"
-  s.description  = <<-DESC
-                   Core iOS client library for the Venmo api
-                   DESC
-
-  s.homepage     = "https://github.com/venmo/VENCore"
+  s.name         = 'VENCore'
+  s.version      = '3.1.2'
+  s.summary      = 'Core Venmo client library'
+  s.description  = 'Core iOS client library for the Venmo api'
+  s.homepage     = 'https://github.com/venmo/VENCore'
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
-  s.author       = { "Venmo" => "ios@venmo.com" }
+  s.author       = { 'Venmo' => 'ios@venmo.com' }
   s.platform     = :ios, '6.0'
-  s.source       = { :git => "https://github.com/venmo/VENCore.git",
-                     :tag => "v#{s.version}"
-  }
+  s.source       = { :git => 'https://github.com/venmo/VENCore.git',
+                     :tag => "v#{s.version}" }
   s.source_files = 'VENCore/**/*.{h,m}'
-  s.dependency 'CMDQueryStringSerialization', '~>0.2.0'
-  s.requires_arc = true
+  s.dependency 'CMDQueryStringSerialization'
```

Comment:
Thinking about it more, I think it's a bad idea to remove @calebd's library if we need query string parsing. This junk is complicated and he has done a fanatic job handling all of the edge cases and writing tests for it. I definitely don't want to maintain that.

Totally down to lock to a version. If we need to update it (assuming we won't often), we can just bump the version of all of our junk.